changelog/fragments/enable-run-bundle-cmd.yaml

```
@@ -0,0 +1,4 @@
+entries:
+  - description: >
+      Enable `run bundle` command and include e2e tests.
```

```suggestion
      Add the subcommand `operato-sdk run bundle` which allows users check their project integration with OLM via the bundle. 
```


The e2e are not relevant to end-users

---

This should be sufficient:
```suggestion
      Added the `run bundle` command.
```